test/appservice/integration/provisionTest.ts

```diff
@@ -3,12 +3,15 @@ import { newTestUser } from "../../integration/clientHelper";
 import { getFirstReply } from "../../integration/commands/commandUtils";
 import { MatrixClient } from "matrix-bot-sdk";
 import { MjolnirAppService } from "../../../src/appservice/AppService";
+import dns from 'node:dns';
 
 interface Context extends Mocha.Context {
     moderator?: MatrixClient,
     appservice?:  MjolnirAppService
 }
 
+dns.setDefaultResultOrder('ipv4first');
+
```

Comment:
needed because apparently node 17+ defaults to IVP6 but GH actions doesn't support IPV6 - see discussion at https://github.com/matrix-org/conference-bot/pull/186/files/85059d8f92ddcfb80657a58f60ef891ce704dfec..723a939d4726aa9de86fae0693137d78674154cb
this caused me a lot of pain until I figured it out...

Reply:
what a nightmare, and weird that it tries to do that...

We should put a similar comment from the conference bot's code here too, for when we inevitably try to remove it by accident. Likewise on the other files.